homeassistant/components/media_player/samsungtv.py

```diff
@@ -171,6 +183,9 @@ def supported_features(self):
 
     def turn_off(self):
         """Turn off media player."""
+        self._end_of_power_off \
```

Comment:
I don't think you need to break this line anymore.

Reply:
You're right, I removed it.